### PR TITLE
Review 01/11/2023

### PR DESCRIPTION
**Pull request by Johannes-Schneider**

### CODE REVIEW

File: exercises/ex4/README.md [hunk 1]
```diff
@@ -12,32 +12,13 @@ Remember, we want to add a goal and add tasks to the goal when a user registers
...
-    @Qualifier(cds.gen.goal.Goal_.CDS_NAME)
-    private CqnService goalService;
-    ```
-
-// the tip below seems to be out of place
-> **Tip:** Treat the package names with caution. Depending on how you name your CDS entities and services as well as how remote services are named there may be naming clashes. 
-> 
-> In our case the generated API class for the SuccessFactors service is similar to the generated entity class `cds.gen.goalservice.Goal_` of our `GoalServcie`. So the fully qualified class name is shown here.
-> 
-> Feel free to rename the `cds.gen.goalservice.Goal_` entity later as an optional exercise.
```
Comment (Johannes-Schneider) [inline exercises/ex4/README.md:32]: I removed this paragraph as I added the corresponding code (commented) to the class already. Without the code added, the application cannot be compiled at an earlier exercise.

Comment (Johannes-Schneider) [reply exercises/ex4/README.md:32]: No, should be everything I think.

Comment (Johannes-Schneider) [reply exercises/ex4/README.md]: Done as part of https://github.com/SAP-samples/teched2023-AD266/pull/17/commits/866204cbbcb3e3296b38c5db2f5cef7d8c7d796b

File: exercises/ex2/README.md [hunk 2]
```diff
@@ -150,7 +152,7 @@ In order for the above code to function at runtime we'll need to provide a **_de
 
 **Further Resources**:
 - [BTP Connectivity: Configuring Destinations in the BTP Cockpit](https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/connectivity-administration?q=Destination%20Service)
-- [SAP Cloud SDK: Using Destinations](https://sap.github.io/cloud-sdk/docs/java/features/connectivity/destination-service)
+- [SAP Cloud SDK: Using Destinations](https://sap.github.io/cloud-sdk/docs/java/v5/features/connectivity/destination-service)
```
Comment (MatKuhr) [inline exercises/ex2/README.md:155]:
```suggestion
- [SAP Cloud SDK: Using Destinations](https://sap.github.io/cloud-sdk/docs/java/features/connectivity/destination-service)
```

Comment (MatKuhr) [reply exercises/ex2/README.md:155]: we are using v4 links on purpose generally

**Review by MatKuhr — APPROVED** [review]

**Pull request merged**